app/models/item.rb

```diff
@@ -3,7 +3,7 @@ class Item < ActiveRecord::Base
   has_many  :items_categories
   has_many  :categories, through: :items_categories
   has_many  :orders_items
-  has_many  :orders, through: :orders_items
+  has_many  :orders, through: :purchases
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.